### DISCUSSION TITLE
HEIF image format support

### DISCUSSION THREAD

**Discussion by justrealmilk** [directus/directus#9104] [Feature Requests]
As it's the default format for iPhone, can HEIF support be enabled some how?

**Comment by benhaynes** [comment]
Is this metadata that would be scraped? Or an actual file format?

**Reply by justrealmilk** [reply]
It's an actual file format/container! It even made [the news](https://www.theverge.com/2020/5/20/21262302/ap-test-fail-iphone-photos-glitch-email-college-board-jpeg-heic) last year

**Comment by rijkvanzanten** [comment]
Sharp already seems to support HEIF right now, so this should be fairly easy to get officially going in Directus as well 👍🏻 Wanna open a PR @justrealmilk?

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
